docs/UserGuide.md

```diff
@@ -31,104 +31,249 @@
 
 > **Command Format**
 > * Words in `UPPER_CASE` are the parameters.
-> * Items in `SQUARE_BRACKETS` are optional.
+> * The order of parameters does not matter.
```

Comment:
btw, also, you need to specify the date and time formats you will support in the application somewhere.